packages/xo-server/src/api/proxy.mjs

```diff
@@ -211,3 +211,13 @@ updateApplianceSettings.params = {
     optional: true,
   },
 }
+
+export function bindLicense({ licenseId, boundObjectId }) {
+  return this.bindLLicense({ licenseId, boundObjectId })
+}
+
+bindLicense.permission = 'admin'
+bindLicense.params = {
+  licenseId: { type: 'string' },
+  boundObjectId: { type: 'string' },
+}
```

Comment:
This belong in `xo-server-xoa`.

Also, is this really necessary, a lot of actions are already available.

Reply:
@Rajaa-BARHTAOUI Is this really necessary, lots of actions are already available.

Reply:
Changes in `xo-server-xoa` have been merged.